pkg/iac-providers/terraform/commons/load-dir.go

```diff
@@ -76,7 +109,7 @@ func (t TerraformDirectoryLoader) LoadIacDir() (allResourcesConfig output.AllRes
 
 	defer t.remoteDownloader.CleanUp()
 
-	if t.nonRecursive {
+	if t.nonRecursive || t.useTerraformCache {
```

Comment:
Trying to understand the reason for using non recursive scanning with terraform cache?